scripts/noshake.json

```
@@ -334,4 +334,4 @@
   "Mathlib.Algebra.Category.Ring.Basic":
   ["Mathlib.CategoryTheory.ConcreteCategory.ReflectsIso"],
   "Mathlib.Algebra.Algebra.Subalgebra.Order":
-  ["Mathlib.Algebra.Module.Submodule.Order"]}}
+  ["Mathlib.Algebra.Module.Submodule.Order"]}}
```

```suggestion
  ["Mathlib.Algebra.Module.Submodule.Order"]}}

```
@digama0, can you make it so that `lake exe shake --ignore` adds the final newline? It's annoying that PRs keep on getting polluted by this whitespace change.